src/Directory.Build.targets

```diff
@@ -61,6 +61,6 @@
   <PropertyGroup>
     <!-- Workaround https://github.com/dotnet/project-system/issues/3670 by setting TargetFramework expicitly -->
     <!-- This property is set late in the project evaluation so other properties don't try to use it. -->
-    <TargetFramework>$(TargetGroup)</TargetFramework>
+    <TargetFramework>$(NuGetTargetMonikerShort)</TargetFramework>
```

Comment:
This allows building/restoring the `ilproj` directly to work (which means you don't have to build all of CoreFX)